src/index.ts

```diff
@@ -1,6 +1,7 @@
 import { colorFromId } from "./utils/utils";
 
 export * from "./components";
+export * from "./beans";
```

Comment:
con questo export non diventano inutili tutti gli export sotto? non sono sicuro ma mi viene il dubbio 🤔

Reply:
L'export si limita ad esportare solo gli enum e le interfacce presenti nella cartella beans quindi non dovrebbe influenzare gli altri export

Reply:
si lo so. intendo che esportandoli tutti in un colpo con questo export, gli altri non dovrebbero servire più